packages/teleport/src/Main/Main.tsx

```diff
@@ -99,6 +107,7 @@ export function Main(props: State) {
       <RouterDOM.Switch>
         <Redirect exact={true} from={cfg.routes.root} to={indexRoute} />
       </RouterDOM.Switch>
+      <BannerList banners={banners} />
```

Comment:
any reason why this banner isn't fixed at the top like it is for our expired license and switchback banner? without it, you can get awkward/confusing double horizontal scrollers (outer scroll and an inner scroll)

Reply:
I'm not able to reproduce this, can you give me some repro instructions?
> awkward/confusing double horizontal scrollers (outer scroll and an inner scroll)

Reply:
visit the audit log page? that should have sufficient data in the table to cause the inner scrolling.

Reply:
Oh I see what you're referring to now. Fixed!

Reply:
I am not seeing this fixed, could you test it together with either a switchback banner or an expired license banner.

Reply:
The switchback & license banners are in e, I need this to land so that I can make the necessary updates there.